README.md

```diff
@@ -8,16 +8,30 @@ Module that provides an interface for a user to view and change product options,
 
 ## Installation / Setup
 
-1. Ensure that the repo is cloned locally for use. Also ensure that MongoDB is downloaded and started, as it is the database that this module uses.
+### Setup -- These steps only need to be completed the first time this module is to be used.
+
+1. Ensure that the repo is cloned locally for use. Also ensure that Cassandra DB is downloaded and started, as it is the database that this module uses.
 1. From the root of this repo, run `npm install` to install dependencies.
 1. Following the installation of all dependencies, run `npm run build` to generate the script bundle that the module will use.
-1. Lastly, while MongoDB is started and available, run `npm run postinstall` to seed the database.
+1. Run `npm run genCSV` to generate two CSV files -- `items.csv` and `reviews.csv`. These will be used to seed our database. Bear in mind that this command is generating at least 30,000,000 results, so it will have to run for a few minutes. It wouldn't be unreasonable to go get a drink or snack during this time. Also be aware that these files are not small! Have at least 4 GB of available hard drive space for them.
```

Comment:
Looks great, very detailed. Love that you suggest getting a drink or a snack lol